src/popover/popover.ts

```diff
@@ -64,26 +65,41 @@ export class NgbPopover implements OnInit, OnDestroy {
    */
   @Input() container: string;
   /**
+   * When set to a truthy value, closes the popover when a click occurs outside of the popover element
+   */
+  @Input() closeOnClickOutside: boolean;
+  /**
    * Emits an event when the popover is shown
    */
   @Output() shown = new EventEmitter();
   /**
    * Emits an event when the popover is hidden
    */
   @Output() hidden = new EventEmitter();
+  /**
+   * Emits an event when a click occurs outside of the popover element.
+   */
+  @Output() clickOutside = new EventEmitter();
```

Comment:
@pkozlowski-opensource What do you think? Should we just keep [closeOnClickOutside]?